Cargo.toml

```diff
@@ -158,6 +159,7 @@ jwt-simple = { version = "0.12.11", default-features = false, features = [
 ] }
 krata-loopdev = "0.0.21"
 lazy_static = "1.5.0"
+log = "0.4"
```

Comment:
This helps us to get more conditional/leveled/structured log output. I think that we can evolve this to use a decent subset of the existing tracing/telemetry crates although I bet it doesn't work 100% out of the box (as this isn't so much our standard long running service). The important part is that we can start using `info!()`/`error!()`/etc in the code.